kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -13,6 +13,7 @@
       class="core-menu-option"
       role="menuitem"
       :class="$computedClass(optionStyle)"
+      :tabindex="link ? false : '0'"
```

Comment:
Exactly the same behavior, but maybe slightly easier to understand the intent?

```suggestion
      :tabindex="link ? null : '0'"
```